handler/verifiable/handler.go

```diff
@@ -0,0 +1,69 @@
+// Copyright © 2023 Ory Corp
+// SPDX-License-Identifier: Apache-2.0
+
+package verifiable
+
+import (
+	"context"
+	"time"
+
+	"github.com/ory/fosite"
+	"github.com/ory/x/errorsx"
+)
+
+const (
+	draftScope         = "userinfo_credential_draft_00"
+	draftNonceField    = "c_nonce_draft_00"
+	draftNonceExpField = "c_nonce_expires_in_draft_00"
+)
+
+type Handler struct {
+	Config interface {
+		fosite.VerifiableCredentialsNonceLifespanProvider
+	}
+	NonceManager
+}
+
+var _ fosite.TokenEndpointHandler = (*Handler)(nil)
+
+func (c *Handler) HandleTokenEndpointRequest(ctx context.Context, request fosite.AccessRequester) error {
+	if !c.CanHandleTokenEndpointRequest(ctx, request) {
+		return errorsx.WithStack(fosite.ErrUnknownRequest)
+	}
+
+	if !request.GetGrantedScopes().Has("openid") {
+		return errorsx.WithStack(fosite.ErrUnknownRequest)
+	}
```

Comment:
This can probably be simplified as you're only checking for 2 scopes. I think `Has` accepts more than one parameter (or there is a function that does that).